sirepo/simulation_db.py

```diff
@@ -946,6 +974,27 @@ def _create_example_and_lib_files(simulation_type):
             f.copy(d)
 
 
+def _file_list_in_schema(schema, file_type):
+    """Relative paths of local and external files of the given load and file type listed in the schema
+    The order matters for javascript files
+
+    Args:
+        schema (pkcollections.Dict): schema (or portion thereof) to inspect
+        type (str): type of the file (css, js, etc.)
+
+    Returns:
+        [str]: combined list of local and external file paths
+    """
+    paths = []
+    for lf_prop in LOADED_FILE_PROPS:
+        paths.extend(
+            map(lambda file_name:
+                _pkg_relative_path_static(file_type + '/' + lf_prop.dir, file_name),
```

Comment:
I have updated 
[DesignHints](https://github.com/radiasoft/pykern/wiki/DesignHints#impedance-matches-data-structures-align-with-algorithms)
to talk about this specific example